installers/pgo-installer/ansible/roles/pgo-preflight/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- fail:
+    msg: "Please specify the a tag: uninstall, uninstall-metrics, install, install-metrics or update"
+  tags: always
+  when: ansible_run_tags[0] == "all"
+
+- name: Check Operating System
```

Comment:
Is this needed since we will know what os we are creating the image

Reply:
Nope. Remove it.